finetuner/tuner/base.py

```diff
@@ -42,13 +42,18 @@ def get_default_miner(self, is_session_dataset: bool) -> BaseMiner:
         """Get the default miner for this loss, given the dataset type"""
 
 
-class BaseTuner(abc.ABC, Generic[AnyDNN, AnyDataLoader, AnyOptimizer]):
+class BaseTuner(abc.ABC, Generic[AnyDNN, AnyDataLoader, AnyOptimizer, AnyScheduler]):
     state: TunerState
 
     def __init__(
         self,
         embed_model: Optional[AnyDNN] = None,
         loss: Union[BaseLoss, str] = 'SiameseLoss',
+        configure_optimizer: Optional[
+            Callable[[AnyDNN], Union[AnyOptimizer, Tuple[AnyOptimizer, AnyScheduler]]]
+        ] = None,
+        learning_rate: float = 1e-3,
```

Comment:
So this is only the "default" learning rate, if the user does not provide an optimizer (in which case the learning rate stays constant). I think renaming this will suffice

Reply:
The point is, `learning_rate` is an argument of `fit` not something that the `tuner` should be adapted to. Same tuner identity should be able to work with different `learning_rates`.